R/spark_compile.R

```diff
@@ -126,7 +126,7 @@ spark_compile <- function(name,
 #'
 #' @keywords internal
 #' @export
-spark_compile_package_jars <- function(package = rprojroot::find_package_root_file(),
+compile_package_jars <- function(package = rprojroot::find_package_root_file(),
```

Comment:
`pkg` might be better as an argument name (for conformance with devtools), but I like using `rprojroot` since it makes it easier to call this function when you might have set the working directory to e.g. the subdirectory of a project.

E.g. if you needed to write a standalone R script that lived somewhere in the bowels of your package sources, this would still do the right thing without extra 'find my package root' work from the user.